mp/src/game/shared/momentum/mom_gamerules.cpp

```diff
@@ -240,6 +240,7 @@ static const char * const g_szWhitelistedCommands[] = {
     "sv_gravity",
     "sv_maxvelocity",
     "sv_airaccelerate",
+	"sv_accelerate",
```

Comment:
You'll want to do this off the bugfix_and_qol branch, it was updated to use a different whitelist approach recently (7e3f384)